src/main/resources/test-script.js

```diff
@@ -0,0 +1,9 @@
+try {
```

Comment:
Can we move this file to test package?

Reply:
Sure, I will move it to `src/integTest/resources` since that one will be used for integration tests with IPFS/blockchain.